signature-aggregator/aggregator/aggregator_test.go

```diff
@@ -0,0 +1,87 @@
+package aggregator
+
+import (
+	"testing"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/message"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"github.com/ava-labs/awm-relayer/peers"
+	"github.com/ava-labs/awm-relayer/peers/mocks"
+	"github.com/ava-labs/awm-relayer/signature-aggregator/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/mock/gomock"
+)
+
+var sigAggMetrics *metrics.SignatureAggregatorMetrics
+var messageCreator message.Creator
+
+func instantiateAggregator(t *testing.T) (
```

Comment:
Should we create a happy-path test here? Otherwise this PR LGTM

Reply:
I think yes, but I'd prefer to defer it to a subsequent PR. I'm actively working on it but would like to see this PR get merged since it's already quite close.